docs/adr/0021-audit-logging.md

```diff
@@ -0,0 +1,144 @@
+* CCIC directed repository information collection rules/requests (e.g., sources,
+queries, data calls) must be implemented/provided within the time frame specified
+in the request.
+
+Standard 2: As required by CMS, raw audit records must be available in an unaltered
+format to the CCIC.
+
+Standard 3: Raw security information/results from relevant automated tools must be
+available in an unaltered format to the CCIC.
+
+From an ARS 3.1 standpoint CMS as an organization is responsible for the review/analysis/reporting
+of audit records. This is accomplished by dual monitoring by the system security
+personnel and the enterprise security monitoring (CCIC). The CCIC provides
+Tier 2 monitoring of the application for defense in depth measures. The primary
+monitoring of review and analysis of the audit logs should be performed by the
+system security personnel (i.e. ISSO).
+
+Security personnel reviews and analyzes information system audit records no less
+often than weekly for indications of inappropriate or unusual activity as defined
+within the Implementation Standards and reports findings to defined personnel
+or roles (defined in the applicable system security plan).
+
+Implementation Standards:
+
+* Standard 1 - Review system records for initialization sequences, logons
+(successful and unsuccessful), errors, system processes, security software
+(e.g., malicious code protection, intrusion detection, firewall), applications,
+performance, and system resource utilization to determine anomalies no less often
+than once within a twenty-four (24) hour period and on demand. Generate alert
+notification for technical personnel review and assessment.
+
+* Standard 2 - Review network traffic, bandwidth utilization rates, alert notifications,
+and border defense devices to determine anomalies no less often than once within
+a twenty-four (24) hour period and on demand. Generate alerts for technical personnel
+review and assessment.
+
+* Standard 3 - Investigate suspicious activity or suspected violations on the information
+system, report findings to appropriate officials and take appropriate action.
+
+* Standard 4 - Use automated utilities to review audit records no less often than
+once every seventy-two (72) hours for unusual, unexpected, or suspicious behavior.
+
+* Standard 5 - Inspect administrator groups on demand but no less often than once
+every fourteen (14) days to ensure unauthorized administrator, system, and privileged
+application accounts have not been created.
+
+* Standard 6 - Perform manual reviews of system audit records randomly on demand
+but no less often than once every thirty (30) days.
+
+### 6) How long do we need to keep logged data
+
+We can keep forever. The organization retains audit records for ninety (90) days
+and archive old records for one (1) year to provide support for after-the-fact
+investigations of security incidents and to meet regulatory and CMS information
+retention requirements.
```

Comment:
It looks like this language is pulled from ARS 3.1 AU-11, but there's other CMS documentation that mentions longer data retention, so we may need to confirm the requirement.